osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -287,6 +275,21 @@ public int NumberOfAvatars
 
         protected virtual Drawable CreateBackground() => new OnlinePlayBackgroundSprite();
 
+        protected virtual IEnumerable<Drawable> CreateBottomDetails() => new Drawable[]
+        {
+            new PlaylistCountPill
+            {
+                Anchor = Anchor.CentreLeft,
+                Origin = Anchor.CentreLeft,
+            },
+            new StarRatingRangeDisplay
+            {
+                Anchor = Anchor.CentreLeft,
+                Origin = Anchor.CentreLeft,
+                Scale = new Vector2(0.8f)
+            }
+        };
```

Comment:
Could still have the pills added in the base `CreateBottomDetails()` method itself rather than in both classes?

```csharp
        protected virtual IEnumerable<Drawable> CreateBottomDetails()
        {
            var pills = new List<Drawable>();

            if (Room.Type.Value != MatchType.Playlists)
            {
                pills.AddRange(new OnlinePlayComposite[]
                {
                    new MatchTypePill(),
                    new QueueModePill(),
                });
            }

            pills.AddRange(new OnlinePlayComposite[]
            {
                new PlaylistCountPill
                {
                    Anchor = Anchor.CentreLeft,
                    Origin = Anchor.CentreLeft,
                },
                new StarRatingRangeDisplay
                {
                    Anchor = Anchor.CentreLeft,
                    Origin = Anchor.CentreLeft,
                    Scale = new Vector2(0.8f)
                }
            });

            return pills;
        }
```

Reply:
Good point.